src/js/messaging/reducers/folders.js

```diff
@@ -27,17 +26,11 @@ const initialState = {
       attributes: {},
       filter: {},
       messages: [],
-      pagination: {
-        currentPage: 0,
-        perPage: 0,
-        totalEntries: 0,
-        totalPages: 0
-      },
-      persistFolder: 0,
+      pagination: {},
```

Comment:
Is it okay for all of the values to be null initially? It looks like some of these attributes may flow through to a couple of components that use it. It only affects the initial render (before folders are fetched and the API populates the pagination attribute, but it still might be visible while the API request is coming back

https://github.com/department-of-veterans-affairs/vets-website/blob/d2bfd141baedaee8f1013ff3b0f9bea03f43e425/src/js/messaging/components/MessageNav.jsx#L39